apps/passport-server/migrations/46_zuconnect_day_passes.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE zuconnect_tickets ADD COLUMN extra_info VARCHAR[] DEFAULT '{}';
```

Comment:
fwiw postgres does support json (use the `jsonb` column type). if you make it be jsonb rather than varchar, that may make it possible to more cleanly query by this data, and to manipulate it as well.

Reply:
Fortunately we don't need to do anything interesting with this, but if we wanted to query on it then I would definitely use JSONB (or join tables).